app/models/export/invoices/extract.rb

```diff
@@ -1,11 +1,14 @@
 module Export
   class Invoices
     module Extract
-      def self.all_relevant
+      def self.all_relevant(date_range = nil)
+        submission_scope = Submission.completed
```

Comment:
So this isn't something to be done as part of this PR, but is this the correct scoping? Or are we meant to be exporting `Export::Submissions::Extract:: RELEVANT_STATUSES`?

Reply:
OK, so I checked the original specs and we're only meant to report invoices and orders for successfully completed submissions.